src/translations/IntlContext.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useContext } from "react";
+import { IntlProvider } from "react-intl";
+import sk from '../compiled-lang/sk.json';
+import en from '../compiled-lang/en.json';
+
+const Context = React.createContext(null);
+
+//TODO waiting for what the array will look like
+const languageMessageData = [
+  {
+    "locale": "en",
+    "message": en
+  },
+  {
+    "locale": "sk",
+    "message": sk
+  }
+]
+
+//TODO I'm waiting for what the array will look like. Then refactoring the code
+const loadLocaleData = (locale) => {
+  for (let i = 0; i < languageMessageData.length; i++){
+    if(languageMessageData[i].locale === locale){
+      return languageMessageData[i].message
+    }
+  } 
+  return en
+};
+
+const localeNavigatorLang = navigator.language.slice(0, 2)
```

Comment:
make this null safe with default first item in languageMessageData 
`navigator?.language?.slice(0, 2) ?? languageMessageData[0]?.locale ?? 'en'`